tests/integrations/test_zk_evm.py

```diff
@@ -51,15 +13,15 @@ class TestZkEVM:
         "params",
         params_execute,
     )
-    async def test_execute(self, zk_evm: StarknetContract, params: dict, request):
+    async def test_execute(self, kakarot: StarknetContract, params: dict, request):
```

Comment:
👍 we should rename zk_evm everywhere in the code, in the filename as well

Reply:
I hesitated to rename the file actually; but I'm not sure where we want to go with the test organisation so left it like that; like I think that `test_erc20` should be a separate file dedicated to the testing of the `ERC20.sol` but this PR was about tooling and not test reorg.